resources/sts/4.10-tag-policies/sts-installer-preflight-checks.json

```diff
@@ -0,0 +1,94 @@
+{
```

Comment:
@feichashao what was this file used for?

Reply:
This was used to workaround the pre-flight check SDA-6233.